accounts/views.py

```diff
@@ -32,6 +34,14 @@ class WelcomeTemplateView(TemplateView, LoginRequiredMixin):
 
 class UserLoginView(LoginView):
     template_name = 'account/login.html'
+    form_class = RememberMeLoginForm
+    expiry_age = 30000000
```

Comment:
Try to write this as something we can know when we see. Something like if it was 1 year `expiry_age = 365 * 24 * 60 * 60`